raiden/tests/unit/test_utils.py

```diff
@@ -0,0 +1,11 @@
+import hashlib
+
+from raiden.utils import crypto
+
+
+def test_privtopub():
+    privkey = hashlib.sha256(b'secret').hexdigest()
```

Comment:
Any reason why you don't use the [`raiden.utils.sha3` function](https://github.com/raiden-network/raiden/blob/master/raiden/utils/__init__.py#L34)?

Reply:
I'll make that change, forgot about the utility function